tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/ConnectionPool.java

```diff
@@ -89,26 +110,6 @@ public static void main( String[] args ) throws SQLException {
       } catch (SQLException e) {
         System.out.println( "Failed to createStatement: " + e);
       }
```

Comment:
Can we keep one example for loading jobs? e.g., "loadMember", or "loadCompany"

Reply:
Example of loading job is in file [.../tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/UpsertQuery.java]
e.g., "loadPost"